pyramid_oereb/contrib/sources/plr_oereblex.py

```diff
@@ -37,13 +37,33 @@ def __init__(self, **kwargs):
         self._oereblex_source = OEREBlexSource(**Config.get_oereblex_config())
         self._queried_lexlinks = {}
 
+    @staticmethod
+    def get_config_value_for_plr_code(url_param_config, plr_code):
+        """
+        Returns the appropriate configuration entry for a plr within a url_param_config section.
+        """
+        for url_param_entry in url_param_config:
+            if url_param_entry['code'] == plr_code:
+                if 'url_param' in url_param_entry:
+                    return url_param_entry['url_param']
+                else:
+                    log.warning("Incorrect configuration: missing url_param for entry {}".format(plr_code))
+                    return None
+        return None
+
     def get_document_records(self, params, public_law_restriction_from_db):
         """
         Override the parent's get_document_records method to obtain the oereblex document instead.
         """
-        return self.document_records_from_oereblex(params, public_law_restriction_from_db.geolink)
+        oereblex_params = None
+        url_param_config = self._oereblex_source._url_param_config
+        if url_param_config:
+            plr_code = self._theme_record.code
+            oereblex_params = self.get_config_value_for_plr_code(url_param_config, plr_code)
```

Comment:
```suggestion
            oereblex_params = DatabaseOEREBlexSource.get_config_value_for_plr_code(url_param_config, plr_code)
```

Would be better no ? As the method is static

Reply:
Yes, thanks